### PR TITLE
Config UI Plugin Install Fixed

### PR DESCRIPTION
**Pull request by krishnaglick**
## Change description

Actionhero Upgraded

### Development

- [x] Application changes have been tested appropriately

### Impact

- [x] Code follows company security practices and guidelines
- [x] Security impact of change has been considered
- [x] Performance impact of change has been considered
- [x] Possible migration needs considered (model migrations, config migrations, etc.)

Please explain any security, performance, migration, or other impacts if relevant:

### Code review

- [x] Pull request has a descriptive title and context useful to a reviewer. Screenshots or screencasts are attached where applicable.
- [x] Relevant tags have been added to the PR (bug, enhancement, internal, etc.)

### CODE REVIEW

**Comment by krishnaglick** [comment]
@evantahler Do you want me to add bluebird to config-ui in this?

File: plugins/@grouparoo/airtable/package.json [hunk 1]
```diff
@@ -31,7 +31,6 @@
   "dependencies": {
     "@grouparoo/app-templates": "0.8.0-alpha.5",
     "@grouparoo/core": "0.8.0-alpha.5",
-    "actionhero": "28.1.2",
```
Comment (evantahler) [inline plugins/@grouparoo/airtable/package.json:34]: Was this change on purpose?

**Comment by evantahler** [comment]
I think Adding bluebird should be it's own change/PR, provided it really does fix the install problems

File: plugins/@grouparoo/airtable/package.json (hunk 1 above)
Comment (krishnaglick) [reply plugins/@grouparoo/airtable/package.json:34]: Yes, since it was in both deps and dev deps I thought that was an accident.

**Review by evantahler — APPROVED** [review]

**Comment by evantahler** [comment]
oh! looks like a few new plugins just got merged in - please rebase and check that Klavio etc are incldued in this update
